umm-spec-lib/src/cmr/umm_spec/models/umm_variable_models.clj

```diff
@@ -145,6 +149,34 @@
   ])
 (record-pretty-printer/enable-record-pretty-printing SamplingIdentifierType)
 
+;; This object describes a store for a variable instance. A variable instance is when the variable
+;; is extracted from the original data files and stored somewhere.
+(defrecord InstanceInformationType
+  [
+   ;; The internet location of the variable instance store.
+   URL
+
+   ;; Describes the format of the URL's data content so that users and applications know how to read
+   ;; and use the content. At this point in time, an industry standard mime-type does not exist, so
+   ;; a mime-type element will not be included. The controlled vocabulary for formats is maintained
+   ;; in the Keyword Management System (KMS):
+   ;; https://gcmd.earthdata.nasa.gov/KeywordViewer/scheme/DataFormat?gtm_scheme=DataFormat
```

Comment:
parameters are not needed, KMS will redirect to this URL if you type in https://gcmd.earthdata.nasa.gov/KeywordViewer/scheme/DataFormat, lets go with shorter and less ugly.

Reply:
This file is automatically generated from the umm-var-json-schema.json file.  We should not make changes to it by hand.  But I can change it in the schema and re-generate the file.

Reply:
I have made the change by updating the schema file and re-generating the model file.